tests/drivers/spi/spi_slave/boards/nrf54h20dk_nrf54h20_common.dtsi

```diff
@@ -60,10 +60,10 @@ dut_spi: &spi130 {
 	pinctrl-names = "default", "sleep";
 	overrun-character = <0x00>;
 	cs-gpios = <&gpio0 10 GPIO_ACTIVE_LOW>;
+	wake-gpios = <&gpio0 2 (GPIO_OPEN_SOURCE | GPIO_ACTIVE_HIGH)>;
```

Comment:
`GPIO_OPEN_SOURCE` is unneeded here and actually a bit misleading (the pin is not configured this way). This flag will be ignored by the driver anyway.
But for the wake mechanism to work, `wake-gpios` property needs to be configured on both sides. If this actually works in the current form, it must be by chance.

Reply:
I will remove GPIO_OPEN_SOURCE from DTS.

I failed to get wakeup gpio working on both sides. I created task NRFX-5836 to prepare sample demonstrating how to use this feature.
In the current form, master prints warning that timeout occurred while waiting for confirmation from slave. However, master ignores timeout and sends data.
When I also set wakeup gpio on slave, test hangs. Slave doesn't see wakeup signal from master and hangs in an infinite loop. I lack skills to investigate this issue.
It may be due to invalid GPIO configuration / missing GPIOTE configuration / synchronization issue (i.e. force master to wait longer for confirmation from slave) / etc.

Reply:
> In the current form, master prints warning that timeout occurred while waiting for confirmation from slave. However, master ignores timeout and sends data.

What's the point in adding this property then?

Reply:
I removed wake-gpios as it has no impact on test result.